doc/nrf/ug_nrf5340.rst

```diff
@@ -81,6 +81,9 @@ Trusted Firmware-M (TF-M) support
 ---------------------------------
 
 You can use Trusted Firmware-M (TF-M) as an alternative to :ref:`secure_partition_manager` for running an application from the non-secure area of the memory.
+Support for TF-M in |NCS| is currently experimental, however it is more robust and complete software offering which will be the default option in the future.
+
+Therefore, if your application does not depend on SPM :ref:`lib_secure_services`, it is recommended to use TF-M.
```

Comment:
I don't think we should "recommend" TF-M here; it is sort of confusing, since we do not add this as default anywhere. I guess the idea here should be (along the following lines)

"if your application does not depend or make use of the secure services developed in SPM, then TFM could replace SPM as the secure firmware component in your application."

@oyvindronningstad @hakonfam agreed?

Reply:
Given the trouble you have to go through currently to get TF-M in a usable state with respect to flash usage, I would say that we should not recommend using it.

Reply:
Ok, not recommending then.